client/pk-console.c

```diff
@@ -1652,9 +1654,17 @@ main (int argc, char *argv[])
 				ctx,
 				NULL);
 
+	// Shrink the progresbar to fit in small spaces i.e. termux, small tmux panes, large font terminals
+	struct winsize w;
+	ioctl(STDOUT_FILENO, TIOCGWINSZ, &w);
```

Comment:
space before bracket, and I think we need to initialize the `w` object with NULL. I think we should only query `ws_col` if the ioctl succeeded too.

Reply:
Fixed the formatting and ioctl invocation logic.   As far as init-ing with NULL, I have no idea what the correct way to do it is (none of the examples I found did it), alas I'm much not a C guy...